Gemfile

```diff
@@ -47,6 +47,8 @@ gem "sidekiq", "~> 6.1"
 
 gem "config", "~> 2.2"
 
+gem "httparty"
```

Comment:
Do we want to add the versions to these gems to keep things consistent?

Reply:
I can do. I generally rely on the `Gemfile.lock` to look after that. I don't see the point pinning unless there's a specific version required for some reason. You're right though, it is inconsistent so I'll update.